app/models/decidim/action_delegator/delegation.rb

```diff
@@ -7,6 +7,9 @@ class Delegation < ApplicationRecord
 
       belongs_to :granter, class_name: "Decidim::User"
       belongs_to :grantee, class_name: "Decidim::User"
+      belongs_to :settings,
+                 foreign_key: "decidim_action_delegator_settings_id",
+                 class_name: "Decidim::ActionDelegator::Settings"
       belongs_to :organization,
```

Comment:
@enricostano suggests we could also model with `belongs_to :votable` and then we can remove the `belongs_to :settings`. This would leave us with

```
Delegation ---> Votable <--- Setting
```

I'll explore this approach.